packages/redux/src/analytics/middlewares/__tests__/bag.test.ts

```diff
@@ -177,32 +187,30 @@ describe('bagMiddleware()', () => {
           coupon,
           discount,
           from,
-          oldQuantity,
           position,
-          productId: mockBagProductId,
+          productId: bagMockData.mockProductId,
           quantity,
-          size: sizes[0].id,
+          size: (sizes as SizeAdapted[])[0]?.id,
```

Comment:
Tip: If you type `sizes` variable on the top of the module, you can avoid those casts elsewhere.